solr/solr-ref-guide/src/learning-to-rank.adoc

```diff
@@ -779,3 +869,7 @@ The feature store and the model store are both <<managed-resources.adoc#managed-
 * "Learning to Rank in Solr" presentation at Lucene/Solr Revolution 2015 in Austin:
 ** Slides: http://www.slideshare.net/lucidworks/learning-to-rank-in-solr-presented-by-michael-nilsson-diego-ceccarelli-bloomberg-lp
 ** Video: https://www.youtube.com/watch?v=M7BKwJoh96s
+
```

Comment:
We've got _"... Contributions for further models, features and normalizers are welcome. ..."_ above, any thoughts on adding "interleaving algorithms" to that list?